azure-pipelines.yml

```diff
@@ -18,66 +18,64 @@ jobs:
 
   # Install NuGet
   - task: NuGetToolInstaller@0
-    displayName: Install NuGet 5.6.0
+    displayName: Install NuGet 6.0
     inputs:
-      versionSpec: 5.6.0
+      versionSpec: 6.0.0
 
   # Install NerdBank GitVersioning
   - task: DotNetCoreCLI@2
     displayName: Install NBGV tool
     inputs:
       command: custom
       custom: tool
-      arguments: install --tool-path . nbgv
+      arguments: install -g nbgv
+
+  # Set Build Version
   - script: nbgv cloud
     displayName: Set NBGV version
 
   # Verify headers
-  - powershell: .\build\build.ps1 -Target Verify
+  - pwsh: build/Update-Headers.ps1 -Verify
```

Comment:
Why the change from `powershell` to `pwsh` here?

Reply:
The script I wrote uses a few features that are only available in PowerShell Core (_v6+_). But feel free to update the script to work with Windows PowerShell (_v5.1_). And then you can use `powershell` task!